### ISSUE TITLE
Reproducing WRN-28-10 (SAM) for SVHN dataset

### ISSUE THREAD

**Issue by akshaydp1995** [davda54/sam#19]
I am trying to reproduce the results for WRN-28-10 (SAM) trained on 10-class classification SVHN dataset (Percentage Error 0.99) - https://paperswithcode.com/sota/image-classification-on-svhn  

I'm able to train WRN-28-10 using https://github.com/hysts/pytorch_wrn (Modified the script to incorporate SAM into it)

I'm achieving a test accuracy of 93%. How can I replicate the SOTA Percentage Error 0.99 for WRN-28-10 (SAM). Which hyperparameters do I use?

Any help is appreciated!!

**Comment by davda54** [comment]
Hi, I'm sorry but I'm not one of the authors and I haven't tried to reproduce the results. Anyway, 93 % seems to be a really low accuracy, you should try to follow the hyperparameters from their paper and also evaluate the baseline model with SGD. To achieve 99 %, you also need to use AutoAugment.

**Comment by davda54** [comment]
Closing, feel free to reopen this issue if you have some questions.
